Anarchist Library.js

```diff
@@ -0,0 +1,455 @@
+	"creator": "Sister Baæ'l",
+	"target": "https://theanarchistlibrary.org/(latest|library|stats/popular|category/topic|category/author|special/index|search)",
+	"minVersion": "7.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2025-10-07 11:18:38"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2025 Dandelion Good and the righteous Anti Toil Theologians at Iliff
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+/*
+***** BEGIN ATTRIBUTION BLOCK *****
+This translator was developed by Dandelion Good.
+
+If you do any work on this translator, please add yourself here <3.
+*/
+
+var urlBase = "https://theanarchistlibrary.org/";
+
+// ToDo: Localization
+var languageNames = new Intl.DisplayNames(['en'], { type: 'language' });
+
+var allAttachmentTypes = {
+	"Plain PDF": ".pdf",
+	"A4 PDF": ".a4.pdf",
+	"Letter PDF": ".lt.pdf",
+	EPub: ".eupb",
+	"Printer-friendly HTML": ".html",
+	LaTeX: ".tex",
+	"Plain Text": ".muse",
+	"Source Zip:": ".zip",
+	Snapshot: "snapshot"
+};
+
+function getListItems(doc) {
+	// todo copied from below
+	let items = {};
+	let results = doc.querySelectorAll('a.list-group-item');
+	for (let i = 0; i < results.length; i++) {
+		items[results[i].href] = ZU.trimInternal(text(results[i], "strong"));
+	}
+	return items;
+}
+
+function getSearchItems(doc) {
+	let items = {};
+	let results = doc.querySelectorAll('a.list-group-item');
+	for (let i = 0; i < results.length; i++) {
+		items[results[i].href] = ZU.trimInternal(text(results[i], "strong"));
+	}
+	return items;
+}
+
+async function doLibraryItem(doc, url = doc.location.href) {
+	// ToDo: get fancier here, allow other types
+	let item = new Zotero.Item('webpage');
+
+	let language = languageNames.of(attr(doc, "html", "lang"));
+
+	// Since connectors don't currently filter these, make a subset of all
+	// possible
+
+	let attachmentTypes = {
+		PDF: allAttachmentTypes["Plain PDF"],
+		EPub: allAttachmentTypes.EPub,
+		LaTeX: allAttachmentTypes.LaTeX,
+		Snapshot: allAttachmentTypes.Snapshot
+	};
+
+	item.accessed = new Date().toString();
+	item.url = url;
+	item.language = language;
+
+	let itemType = attr(doc, '[property~="og:type"]', 'content');
+	const tagNodeList = doc.querySelectorAll(`[property~="og:${itemType}:tag"]`);
```

Comment:
We generally avoid `const` for things that aren't static constants, so:

```suggestion
	let tagNodeList = doc.querySelectorAll(`[property~="og:${itemType}:tag"]`);
```